pi5/pinctrl_utils.go

```diff
@@ -12,41 +12,58 @@ import (
 	"strings"
 
 	mmap "github.com/edsrzf/mmap-go"
-	"github.com/pkg/errors"
 )
 
-// rangeInfo represents the info provided in the ranges property of a device tree. It provides a mapping between // registers in the child address space to the parent address space.
+const gpioName = "gpio0"
+const gpioMemPath = "/dev/gpiomem0"
+const dtBaseNodePath = "/proc/device-tree"
+
+// rangeInfo represents the info provided in the ranges property of a device tree. It provides a mapping between addresses in the child address space to the parent address space.
```

Comment:
I suspect you've documented this elsewhere, but in here, describe how to interpret the data of a `rangeInfo`. How do you use these three uint64's to have a mapping between the address spaces?